db/clean-schema.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Adevinta
+
+docker run -v $PWD/pkg/migrations:/migrations --network host migrate/migrate \
```

Comment:
This is not exactly the same as flyway clean, we should evaluate if we need it, and perhaps replace with a psql command to delete all the schema objects.

Reply:
Replaced with psql drop schema, create schema